django_project/changes/templates/version/detail-content.html

```diff
@@ -78,9 +78,9 @@ <h2 class="text-muted" id="category-{{ row.category.id }}">
       {% if entry.approved %}
         <div class="row">
           <div class="{% if entry.image_file %}col-md-7{% else %}col-md-12{% endif %}">
-            <h3 id="feature-{{ entry.id }}">
+            <h3 id="{{ entry.id }}">
               {% if not rst_download %}
-                <a href="#{{ entry.id }}">#</a>
+                <a href="{{ request.path }}#{{ entry.id }}">#</a>
```

Comment:
and then just add feature- in before the entry id here. I provides better semantics.